hack/chains/deploy_local_rekor.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+#
+# A wrapper for deploying rekor into a local cluster.
+# Uses the deploy_local_rekor.tpl file, which has a placeholder for
+# {domain} which is replaced by getting the domain from the openshift
+# cluster. 
+
+# Make sure we have our template file
+if [[ ! -f $PWD/deploy_local_rekor.tpl ]];then
+  echo "Template file 'deploy_local_rekor.tpl' not found in this directory "; exit 1 
+fi
+
+# Test if we have the argocd cli command:
+if ! command -v argocd &> /dev/null; then
+  echo "The argocd CLI is not installed. Please install and try again."
+  echo "See https://argo-cd.readthedocs.io/en/stable/getting_started/#2-download-argo-cd-cli for details"
+  exit 1
+fi
+
+# Get our domain from the ingresses config of the cluster
+domain=$( kubectl get ingresses.config.openshift.io cluster -o json|jq -r ".spec.domain" )
+
+# Set the template input file
+input_file="$PWD/deploy_local_rekor.tpl"
+
+# Set the file we're writing our template output to
+# TODO: Determine if we even need a file or can just use string
+deployment_file=$(mktemp --tmpdir deploy_local_rekor_XXXXXXX.yaml)
+
+# If we're writing temp files, let's trap sigs and remove the file
+# when the script exits
+trap "rm -f $deployment_file" 0 2 3 15
+
+# sed is awesome...
+sed -e "s/\${domain}/$domain/" $input_file > $deployment_file
```

Comment:
It's crude but should do the job.

I once used `envsubst` but I don't think it's particularly better.

I do wonder if argocd has a built in way to do this, but that can be a question for another day, rather than a blocker for this PR IMO.